### PR TITLE
docs: add ONVIF-noncompliant camera list, Snap subsection

### PR DESCRIPTION
**Pull request by MonicaisHer**
Signed-off-by: MonicaisHer <beilimengmeng@gmail.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-camera-go/blob/master/.github/CONTRIBUTING.md

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## New Imports

- [ ] Yes
- [X] No

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -13,6 +13,22 @@ should be compatible.
 * Bosch NBN-832V-IP
 * Hikvision DS-2CD2342WD-I
 
+Tested ONVIF-noncompliant cameras list:
```
Comment (siggiskulason) [inline Readme.md:16]: I would change this to something like:

Only those ONVIF-compliant cameras which support the "onvif_snapshot" method are fully compliant with the EdgeX Camera device service. They include the following cameras which have been tested with EdgeX:
- SV3C Wireless WiFi Security Camera

The following cameras are non-compliant:
- TP-Link Tapo C110

File: Readme.md [hunk 2]
```diff
@@ -13,6 +13,22 @@ should be compatible.
...
+* TP-Link Tapo C110
+
+## Running EdgeX Device Camera with Snap package
```
Comment (siggiskulason) [inline Readme.md:19]: Change this to "Running the device-camera service as a snap"

File: Readme.md [hunk 3]
```diff
@@ -13,6 +13,22 @@ should be compatible.
...
+
+EdgeX Device Camera is also available as a Snap package. The easy and secure way to install Device Camera on Linux is using Snap:
```
Comment (siggiskulason) [inline Readme.md:21]: Change this to "The device service is also available as a snap. Install the snap with the following command:"

**Review by siggiskulason — CHANGES_REQUESTED** [review]
Thanks for adding this. I made a just a couple of comments. Could you make those changes if you are ok with them and then update your PR. You also need to rebase your branch to pull in changes made upstream so you can do something like

```
# first to rebase
git remote add upstream https://github.com/edgexfoundry/device-camera-go.git
git pull --all
git rebase upstream/main

# then make your changes and push them, amending your previous commit:
git add Readme.md
git commit --amend
git push -f origin
```

File: Readme.md (hunk 1 above)
Comment (MonicaisHer) [reply Readme.md]: Thank you for your suggestion! Changed.

File: Readme.md (hunk 2 above)
Comment (MonicaisHer) [reply Readme.md]: Changed

File: Readme.md (hunk 3 above)
Comment (MonicaisHer) [reply Readme.md]: Changed

**Review by siggiskulason — APPROVED** [review]
LGTM

**Pull request merged**